uiowa/uiowa
# update media_thumbnails_video

Relates to https://github.com/uiowa/uiowa/issues/6381
If you upload a video for a banner, does it create a still image for paused by default banners?

Just some PHP and Drupal compatibility fixes. One of which we were pinned at.
https://git.drupalcode.org/project/media_thumbnails_video/-/compare/8.x-1.0-alpha2...2.0.0?from_project_id=62163&straight=false

# To Test

- `ddev composer install`
- Sync a v3 site. Add a banner to layout. For the media upload/select a video file. Set autoplay to false.
- On render an image from the first second should display.

Test video (right-click save-as..)
https://github.com/uiowa/uiowa/assets/4663676/e6382cd6-9f8e-444b-a664-c8c027b38f9c
